src/main/kotlin/racingcar/domain/RacingGameReport.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain
+
+class RacingGameReport(
+    lapReports: List<LapReport> = listOf()
```

Comment:
```suggestion
    lapReports: List<LapReport> = emptyList()
```
emptyList로 빈 리스트를 표현할 수 있어요